contrib/test/integration/vars.yml

```diff
@@ -12,7 +12,7 @@ build_kata: False
 cgroupv2: False
 
 critest_mirror_repo: quay.io/crio
-cri_tools_git_version: "v1.27.0"
+cri_tools_git_version: master
```

Comment:
any way this can be converted to "main"?

Reply:
Yes, when we agree on https://github.com/kubernetes-sigs/cri-tools/issues/1100